runtime/src/lib.rs

```diff
@@ -103,10 +98,11 @@ pub struct Drivers<'a> {
     /// Ecc384 Engine
     pub ecc384: Ecc384,
 
-    pub fht: &'a mut FirmwareHandoffTable,
+    pub persistent_data: PersistentDataAccessor,
 
-    /// ImageManifest for the currently running image
-    pub manifest: &'a ImageManifest,
+    /// A copy of the ImageHeader for the currently running image
+    // TODO: Use persistent_data instead
+    pub manifest: ImageManifest,
```

Comment:
I think this can just be done in this PR right? There are only a couple places that use the manifest and they all look straightforward for using `persistent_data` instead.

Reply:
Done. FWIW, for first pass I was just trying to get rid of all the unsafe code.